### DISCUSSION TITLE
История версий в Code

### DISCUSSION THREAD

**Discussion by and7ey** [sberdevices/salute-issues#178] [SmartApp Code, SmartApp Graph, SmartApp API, Assistant Client]
Где можно увидеть историю версий в Code?
Что значит неработающая ссылка "Код" в Code? Не она ли должна открывать историю версий?

**Comment by Genenko-Svetlana** [comment]
Сейчас данный функционал отсутствует, но его добавление включено в беклог (1 квартал 2021)

**Comment by Genenko-Svetlana** [comment]
Возвращаем в работу.

**Comment by Genenko-Svetlana** [comment]
@and7ey По кнопке "История" можно увидеть историю версий
